extensions/amp-ad-exit/0.1/test-e2e/test-amp-ad-exit.js

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {RequestBankE2E} from '../../../../build-system/tasks/e2e/request-bank';
```

Comment:
Is `RequestBankE2E` specific to `amp-ad-*` or can it be used in other e2e tests?

Reply:
It is general, not limited to ads.

Reply:
Ok, in that case can it be initialized through `describes-e2e` so every test doesn't have to import this?

Reply:
@estherkim Good idea. Updated.